src/Prism.Core/Navigation/IDestructible.cs

```diff
@@ -1,7 +1,13 @@
 ﻿namespace Prism.Navigation
 {
+    /// <summary>
+    /// Interface for destuctible objects 
+    /// </summary>
     public interface IDestructible
     {
+        /// <summary>
+        /// Classic destructor for object to be called on destruction
+        /// </summary>
```

Comment:
This provides a spot to cleanup your View/ViewModel for any resources it has.

Reply:
Modified comments to reflect better description